packages/contracts-bedrock/src/asterisc/RISCV.sol

```diff
@@ -0,0 +1,1652 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { IPreimageOracle } from "src/cannon/interfaces/IPreimageOracle.sol";
+import { IBigStepper } from "src/dispute/interfaces/IBigStepper.sol";
```

Comment:
Is https://github.com/ethereum-optimism/asterisc/blob/master/rvsol/src/RISCV.sol the correct upstream file? It looks different than this one, e.g. I see that one has `pragma solidity ^0.8.13;` and only one import, whereas here we have a different pragma and two imports

Reply:
See https://github.com/ethereum-optimism/asterisc/pull/93